### ISSUE TITLE
Whole validation accuracy using provided model_110.ckpt-79999 is extremely low

### ISSUE THREAD

**Issue by iamlotus** [wenxinxu/resnet_in_tensorflow#25]
I try to test accuracy on the whole validation set using the provided ckpt, so I modify cifar10_train.py like below
```maybe_download_and_extract()
# Initialize the Train object
train = Train()
# Start the training session
# train.train()

validation_array, validation_labels = read_in_all_images([vali_dir],
                                                         is_random_label=VALI_RANDOM_LABEL)
predictions=train.test(validation_array)
vali_accu=np.mean((np.argmax(predictions,1)==validation_labels.astype(int)).astype(float))
print 'total accu on vali is %f'%vali_accu
```
But the result is  **extremly low**  
> total accu on vali is 0.334300

Then I trained my owner checkpoint from scratch, and got a much better accuracy
> total accu on vali is 0.9161000

 Is there anything wrong during my test process?

**Comment by huiofficial** [comment]
I got the same result, and I still training my own model.
